src/Compute/Compute.Test/LiveTests/TestLiveScenarios.ps1

```diff
@@ -0,0 +1,117 @@
+Invoke-LiveTestScenario -Name "Creates a virtual machine." -Description "Test create new VM" -ScenarioScript `
+{
+    param ($rg)
+
+    $rgName = $rg.ResourceGroupName
+    $name = New-LiveTestResourceName
+
+    ## VM Account
+    # Credentials for Local Admin account you created in the sysprepped (generalized) vhd image
+    $VMLocalAdminUser = "LocalAdminUser"
+    $VMLocalAdminSecurePassword = ConvertTo-SecureString "Password" -AsPlainText -Force
+    ## Azure Account
+    $LocationName = "westus"
+    # This a Premium_LRS storage account.
+    # It is required in order to run a client VM with efficiency and high performance.
+    $StorageAccount = "Mydisk"
+
+    ## VM
+    $OSDiskName = "MyClient"
+    $ComputerName = "MyClientVM"
+    ##$OSDiskUri = "https://Mydisk.blob.core.windows.net/disks/MyOSDisk.vhd"
+    ##$SourceImageUri = "https://Mydisk.blob.core.windows.net/vhds/MyOSImage.vhd"
+
+    # Modern hardware environment with fast disk, high IOPs performance.
+    # Required to run a client VM with efficiency and performance
+    $VMSize = "Standard_DS3"
+    $OSDiskCaching = "ReadWrite"
+    $OSCreateOption = "FromImage"
+
+    ## Networking
+    $DNSNameLabel = "mydnsname" # mydnsname.westus.cloudapp.azure.com
+    $NetworkName = "MyNet"
+    $NICName = "MyNIC"
+    $PublicIPAddressName = "MyPIP"
+    $SubnetName = "MySubnet"
```

Comment:
The names DNSNameLable, NetworkName, NICName, PublicIPAddressName, SubnetName all need to be generated using New-LiveTestResourceName. If exception occurs, the cmdlets will run again which prompts for user action. This will cause the pipeline stuck.